src/main/scala/com/codacy/client/bitbucket/client/BitbucketClient.scala

```diff
@@ -78,6 +78,14 @@ class BitbucketClient(key: String, secretKey: String, token: String, secretToken
     value
   }
 
+  def postForm[T](request: Request[T], values: Map[String, Seq[String]])(implicit reader: Reads[T]): RequestResponse[T] = {
```

Comment:
i already talked to pedro. i just don't like the names postForm and postJson. i don't mind hiding the general method but i'd really prefer if the other two were both named post with different signatures